### ISSUE TITLE
extend autohostmanager

### ISSUE THREAD

**Issue by 6uoW** [springlobby/springlobby#478]
someone is elected tempadmin and his interface controls the server.
That needs cooperation of the lobby maker and the server maker.

so no more trouble setting start types, boxes and all.

you would need some sort of protocol that permits the server recieve commands from the lobby but i think is stuff is already there.

thank you good luck

**Comment by abma** [comment]
what? i don't understand...

**Comment by 6uoW** [comment]
you know the servers right? they accept commands like !starboxe 10 10 2

or !map azure

or !starts random

or !gamemode koth or ......

but its a pain in the ass to type such commands every time

and why, if we have a USER INTERFACE that does simplify the taks.

the only thing missing, is the lobby client broadcasting the commands to
the server so it does the box in the right place or change that pesky
setting like random starts or whatever


did i make myself clear?


in other words, you make boxes or change settings in the gui of the lobby
client, and the server gets configured (there is where the protocol comes
in play i guess),

the TEMPADMIN command (now known as !boss), is for determining whos gui
will manage the server.

questions? is it clear now?

**Comment by abma** [comment]
hmm... this code already exists:

https://github.com/springlobby/springlobby/blob/master/src/autohostmanager.cpp#L116

but only very few commands are "translated" yet...

**Comment by 6uoW** [comment]
Someone toldme that is the way relay hosts work.
just we use autohosts much more.
(i think relayhosts are broken atm)

**Comment by abma** [comment]
relayhosts should work atm! if not, please create a new issue.
